test/metabase/db/test_util.clj

```diff
@@ -0,0 +1,37 @@
+(ns metabase.db.test-util
+  (:require [clojure.java.jdbc :as jdbc]
+            [clojure.test :refer :all]
+            [metabase.test :as mt]
+            [potemkin :as p]
+            [pretty.core :as pretty]))
+
+(p/deftype+ ClojureJDBCSpecDataSource [jdbc-spec]
+  pretty/PrettyPrintable
+  (pretty [_]
+    (list `->ClojureJDBCSpecDataSource jdbc-spec))
+
+  javax.sql.DataSource
+  (getConnection [_]
+    (jdbc/get-connection jdbc-spec))
+
+  (getConnection [_ _user _password]
+    (throw (UnsupportedOperationException. "Use (.getConnection this) instead."))))
+
+(alter-meta!
+ #'->ClojureJDBCSpecDataSource
+ assoc
+ :arglists   '(^javax.sql.DataSource [jdbc-spec])
+ :deprecated :true
+ :doc "Return a [[javax.sql.DataSource]] for a [[clojure.java.jdbc]] spec. DEPRECATED -- this is only provided for
```

Comment:
nit: align value here?  It is for `:arglists` above.

Reply:
fixed